packages/shared/src/components/filters/ContentTypesFilter.tsx

```diff
@@ -4,27 +4,24 @@ import { useAdvancedSettings } from '../../hooks/feed';
 import useFeedSettings from '../../hooks/useFeedSettings';
 
 export function ContentTypesFilter(): ReactElement {
-  const { advancedSettings, isLoading } = useFeedSettings();
+  const { videoSetting, isLoading } = useFeedSettings();
```

Comment:
Just moved inside useFeedSettings so the logic could be reused in content type onboarding